File: java/NpowerN.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
    	int Number, Counter = 1, Result = 1, Multiple = 1;
    	Scanner input = new Scanner(System.in);
    	System.out.print("Enter the number how many times you want to multiply the same number with itself:");
    	Number = input.nextInt();
    	for(Counter = 1; Counter <= Number; Counter++)
    	{
    	  Multiple = Number * Multiple;
    	}
    	System.out.print(" Multiplicating " + Number + " with " + Number + " times gives " + Multiple + ".");
    }
}

File: python/NpowerN.py
from __future__ import print_function
Number = input("Enter the number how many times you want to multiply the same number with itself:")
Counter = int(1)
Result = int(1)
print ("Multiplying {0} with {1} times gives".format(Number , Number), end = ' ');
for Counter in range(1, int(Number)+1, 1):  
    Result = Result * int(Number)
print("{0}." .format(Result))

 